content/docs/guidelines/best-practices/checklist/index.md

```diff
@@ -1,28 +1,225 @@
 +++
 title = "The checklist"
 description = ""
-author = "nathan"
+author = "sally"
```

Comment:
We'll need to add author data for you, a file like this: https://github.com/GDQuest/website/blob/master/data/authors/johnny.toml

Could you send me that or create such a file in data/authors? It should be named `sally.toml` in this case.

Reply:
Ok, you can change the author name to Nathan for now.

Reply:
I changed it back :)